ppr-ui/src/store/state/state-model.ts

```diff
@@ -321,10 +326,11 @@ export const stateModel: StateModelIF = {
         region: '',
         country: '',
         postalCode: ''
-      },
+      } as AddressIF,
+      emailAddress: '',
       phoneNumber: '',
-      emailAddress: ''
-    },
+      phoneExtension: ''
+    } as SubmittingPartyIF,
```

Comment:
Is this required in the model? Shouldn't this be on our stateModel interface?

Reply:
You mean the use of IF here? I will remove.

Reply:
Removed.